glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt

```diff
@@ -389,6 +389,11 @@ class GleanTest {
 
         val server = getMockWebServer()
 
+        resetGlean(getContextWithMockedInfo(), Glean.configuration.copy(
```

Comment:
Why is this needed? Is it because of glean handles?

Reply:
Yes, pings are registered through the FFI, but if we reset Glean we also clear the object on the Rust side, at which point we don't have the info around to re-register pings. (technically we _could_ have the data around if we put it into the `pingTypeQueue`, but when I tried that the simple way I ran into multi-threading problems)